Formula/lxc.rb

```diff
@@ -21,11 +21,8 @@ class Lxc < Formula
   depends_on "go" => :build
 
   def install
-    ENV["GOPATH"] = buildpath
     ENV["GOBIN"] = bin
-    ENV["GO111MODULE"] = "auto"
 
-    ln_s buildpath/"_dist/src", buildpath/"src"
     system "go", "install", "-v", "github.com/lxc/lxd/lxc"
```

Comment:
This should be possible now
```suggestion
    system "go", "build", *std_go_args
```

Reply:
I don't think so, wouldn't this build everything in the repository rather than just the one command that works on MacOS?

Reply:
oh, I see `*std_go_args ./lxc` may do it, let's try that